mono_repo/test/mono_repo_test.dart

```diff
@@ -6,8 +6,14 @@ import 'dart:io';
 
 import 'package:test/test.dart';
 import 'package:test_process/test_process.dart';
+import 'package:mono_repo/mono_repo.dart';
 
 void main() {
+  test('commands', () {
+    expect(commands.map((c) => c.name),
```

Comment:
I really don't think this test adds any value. The main value that I think you're going for is that we don't **forget** to add a command to `commands`. But this won't actually catch that case. It will only catch the case that we **remember** to add the command to `commands`, but forget to update the test as well.